python/surf/protocols/clink/_ClinkChannel.py

```diff
@@ -194,11 +194,24 @@ def __init__(
         # Check if serial interface defined
         if serial is not None:
 
+            # Check for BaslerAce camera
+            if (camType=='BaslerAce'):
+
+                # Override defaults
+                self.BaudRate._default = 9600
+
+                # Add the device
+                self.add(surf.protocols.clink.UartBaslerAce(
+                    name   = 'UartBaslerAce',
+                    serial = serial,
+                    expand = False,
+                ))
+
             # Check for OPA1000 camera
-            if (camType=='Opal1000'):
+            elif (camType=='Opal1000'):
 
```

Comment:
A better way to do this is to pass the class itself in to the `camType` attribute.
Then you can get rid of all the `ifelse` logic and just have:
```python
self.BaudRate._default = camType.DEFAULT_BAUD_RATE 
self.add(camType(
    serial=serial,
    expand=False))
```

You'd probably want to rename `camType` to `camClass`.
And maybe it would break a lot of existing code.

Reply:
That will break existing code

Reply:
We can punt on it for now.